core/app/models/spree/variant.rb

```diff
@@ -213,12 +230,39 @@ def option_value(opt_name)
       option_values.detect { |o| o.option_type.name == opt_name }.try(:presentation)
     end
 
+    # Returns an instance of the globally configured variant pricer class for this variant.
+    # It's cached so we don't create too many objects.
+    #
+    # @return [Spree::Variant::Pricer] The default pricer class
+    def pricer
+      @pricer ||= Spree::Config.variant_pricer_class.new(self)
+    end
+
+    # Chooses an appropriate price for the given pricing options
+    #
+    # @see Spree::Variant::Pricer#price_for
+    # @param [Spree::Config.pricing_options_class] An instance of pricing options
+    # @return [Spree::Money] The chosen price as a Money object
+    delegate :price_for, to: :pricer
```

Comment:
Neat!

My only concern here is that `price_for` reads a lot like `price_in`, despite being very different (`price_in` returns a `Price`, `price_for` returns a `Money`). I'm not sure I have a better suggestion, though, as out `*amount*` methods usually just return a number.